content/english/ml-machine-learning/01-regression.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "What is regression?"
+description: "Make computers learn to predict outcomes."
+prereq: "Python"
+icon: ""
+draft: false
+weight: 1
+---
+
+# What is regression?
+Regression is a technique to model the relationship between a feature (independent variables) and a prediction (dependent variables). It helps us understand how the value of a dependent variable is changing based on the value of the independent variable. When applied properly it helps us predict values 😯!
+
+|![Regression](../resources/regression.png)|
+|:--:|
+|Regression|
+
+Regression is a powerful technique that lets us find a line or a curve that fits the data we have. BBy doing regression, we can create or reuse `mathematical models` that show how the dependent and independent variables are related. [Mathematical models](https://www.youtube.com/watch?v=xHtsuOB-TPw) are useful because they help us understand a system and make predictions based on the system's variables. Sometimes, the mathematical model we choose may not match the data well, so we have to look for a better model that can capture the patterns we see.
+
+## Example 1: How do we know if it is going to rain?
+Whenever you use your phone, watch a news cast, or just ask in the internet "What is the chance of rain today?", mathematical models are used by meteorologists to predict if there will be the possibility of rain. You may wonder,  What `variables` are taken into consideration when running those mathematical models? Is the smell of rain enough?
+
+{{% expand "**What variables should you consider in order to know if it is going to rain?**"%}}
+
+- Temperature &rarr; Is it hot, is it cold or is it on the sweet spot?
+- Altitude &rarr; Depending on how high you are relative to the ground places can become cooler or warmer.
+- Location &rarr; Where you are matters. (e.g: Forest, Beach, Mountain Range, Desert, etc) 
+- Humidity &rarr; Do you feel that the air is heavy? Is there enough water in the air for it to rain?
+- Time of the Year &rarr; What month is it? What season are we currently on?
+- And many many more!
+
+There are many more variables to consider whenever we think there is a chance of rain. At the end of the day the "**chance of rain**" is the dependent variable while the other properties we have mentioned above are the independent variables. The chance of rain is dependent on the values of the other variables mentioned.
+{{% /expand %}}
+
+### Exercise 1
+Suppose that someone wants to know the type of shapes they have based on the area of the shape. 
+
+1. Is that even possible? Can someone know what shape they have based on the area?
+{{% expand "**Click to show answer**" %}}  
+***No, as you can quickly guess no relation exists between the **area** and the type of shape.***
+
+For example, a square of side length 3 has an area of 9.
+
+(3 x 3 = 9)
```

Comment:
Could help to add pictures/diagrams/visuals of a square with this length/area, same for the triangle & so on. I think this will make all the math be less scary to kids if they can see the differences as well